examples/simple-example/simple-example-core/src/main/java/io/miragon/miranum/integrations/example/application/service/FireAndForgetService.java

```diff
@@ -11,6 +11,6 @@ public class FireAndForgetService implements FireAndForgetUseCase {
     @Override
     @Worker(type = "fireAndForget")
     public void fireAndForget(final SendMessageCommand message) {
-        log.info("Received message: " + message.getName());
+        log.info("Received message: " + message.getContent());
```

Comment:
It is a bit confusing tbh that this is called content and not message. It is only used as a message to correlate to during the whole example

Reply:
Personally, I found name a bit irritating. I thought a message usually needs to have some content. A message can have a name and content though 🤔

Reply:
Yess your right, I just thought that we just use the content as the message name. In this case it is a MessageCorrelationMessage and not a general message I guess. It can have content as well, but here it does not. It is just an example, so I'm not too concerned about the naming here. Choose what you think is better understand